shpc/tests/test_client.py

```diff
@@ -338,3 +336,41 @@ def test_add(tmp_path, module_sys, remote):
         client.get("dinosaur/salad:latest")
     client.install("dinosaur/salad:latest")
     assert client.get("dinosaur/salad:latest")
+
+
+@pytest.mark.parametrize(
```

Comment:
You are so good at writing tests! I'm bad I always do this last. I admire that.

Reply:
Honestly, I'm doing it here because I saw you doing it. Peer pressure ;) !